packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -123,10 +123,20 @@ export interface IGarbageCollectionSnapshotData {
 	deletedNodes: string[] | undefined;
 }
 
+/**
+ * Contains the necessary information to allow DDSes to do incremental summaries
+ */
+export interface IIncrementalContext {
```

Comment:
`IIncrementalContext` -> `IIncrementalSummaryContext` to indicate this is a context for summary.